lib/cloud_controller/dea/pool.rb

```diff
@@ -9,6 +9,7 @@ def initialize(config, message_bus)
         @message_bus = message_bus
         @percentage_of_top_stagers = (config[:placement_top_stager_percentage] || 0) / 100.0
         @dea_advertisements = {}
+        @min_candidate_stagers = (config[:minimum_candidate_stagers] ? config[:minimum_candidate_stagers] : 5)
```

Comment:
You're right, that's the better approach.  Be pushing up a fresh commit shortly.